### PR TITLE
refresh and simplify with new parent

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <olamy@apache.org>

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (joakime) [inline .github/dependabot.yml:10]: We should specify that these only run for the `main` branch.

File: pom.xml [hunk 2]
```diff
@@ -1,95 +1,75 @@
...
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>5.1.9</version>
+        <extensions>true</extensions>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
+          <instructions>
+            <Bundle-SymbolicName>org.eclipse.jetty.schemas;singleton:=true</Bundle-SymbolicName>
+            <Bundle-Name>Jetty Servlet Schemas</Bundle-Name>
+            <Bundle-RequiredExecutionEnvironment>JavaSE-1.7</Bundle-RequiredExecutionEnvironment>
```
Comment (joakime) [inline pom.xml]: Shouldn't this be the same as `jdk.version.minimum` ?

File: pom.xml [hunk 3]
```diff
@@ -1,95 +1,75 @@
...
+            <_nouses>true</_nouses>
+            <Export-Package>javax.servlet.resources;version="4.0",
+              javax.servlet.jsp.resources;version="4.0",
+              jakarta.servlet.resources;version="5.0",
```
Comment (joakime) [inline pom.xml:49]: Isn't Servlet 5.0 JDK 11 ?

**Review by joakime — CHANGES_REQUESTED** [review]
Looking good, just some tweaks left.

File: .github/dependabot.yml (hunk 1 above)
Comment (olamy) [reply .github/dependabot.yml:10]: not needed. this will run only on default branch as this repo have only one branch

File: pom.xml (hunk 2 above)
Comment (olamy) [reply pom.xml]: no real idea. This hasn't been modified but sounds good to upgrade this value

File: pom.xml (hunk 3 above)
Comment (olamy) [reply pom.xml:49]: yes. but this pom change is conservative with the current setup.
we can definitely mix with upgrade as well. 
But this project contains from web-app_2_2.dtd to web-app_5_0.xsd so the spec version is larger ;)

**Pull request merged**